resources_pages/install_ds_stack_mac.md

```diff
@@ -376,26 +383,28 @@ and try inserting the operators by pressing `Alt` + `-` or `Shift` + `Command` +
 
 We will install the lightest possible version of LaTeX and it's necessary packages as possible so that we can render Jupyter notebooks and R Markdown documents to html and PDF. If you have previously installed LaTeX, please uninstall it before proceeding with these instructions.
 
-First, open RStudio and run the following commands to install the `tinytex` package and setup `tinytex`:
+First, run the following command to make sure that `/usr/local/bin` is writable:
 
-```
+>```shell
+> sudo chown -R $(whoami):admin /usr/local/bin
+>```
+
+> **Note:** You might be asked to enter your password during installation.
+
+Now open RStudio and run the following commands to install the `tinytex` package and setup `tinytex`:
+
+```R
 install.packages('tinytex')
 tinytex::install_tinytex()
 ```
 
-> Note: You might be asked to enter your password during installation.
-> If you see an error message towards the end of the installation
-> telling you that `/usr/local/bin` is not writeable,
-> you will need to open a terminal and run the following two commands before proceeding:
->
->```
-> sudo chown -R $(whoami):admin /usr/local/bin
-> ~/Library/TinyTeX/bin/x86_64-darwin/tlmgr path add
->```
+```bash
+~/Library/TinyTeX/bin/*/tlmgr path add
```

Comment:
In the old instructions we recommend running this right after the `chown` command. Is that not needed? Did you need to run this at all?

Reply:
No! I didn't need it. What is the purpose of this command?

Reply:
It seems to be mostly for documentation related things https://www.tug.org/texlive/doc/tlmgr.html#path. We might be able to remove it since it shouldn't be needed when using Rmarkdown (although maybe it gives issues with Jupyter if we don't have it, but that seems to be fine for you) https://yihui.org/tinytex/#maintenance.

Reply:
So let's keep it for now?

Reply:
Yeah, let's just keep it and if I have time I will explore and remove it (probably wont)